pkgs/by-name/de/dell-bios-fan-control/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  stdenv,
+  fetchgit,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "dell-bios-fan-control";
+  version = "20220119"; # upstream does not use versioning - use date of last commit
```

Comment:
The version should be `0-unstable-20220119` per https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning

Reply:
nit: if it was date of commit, per versioning it would be `0-unstable-2022-01-19`, but we're splitting hairs at this point.